bin/web-ext.js

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+import webExt from '../lib/main.js';
+
+const absolutePackageDir = path.join(
+  path.dirname(fileURLToPath(import.meta.url)),
+  '..'
+);
```

Comment:
I think that that the new way would be to use URL instead of path, something like `fileURLToPath(new URL('..', import.meta.url))`, and you'd have the full path.